lms/djangoapps/course_home_api/toggles.py

```diff
@@ -4,39 +4,25 @@
 
 from edx_toggles.toggles import LegacyWaffleFlagNamespace
 
-from lms.djangoapps.experiments.flags import ExperimentWaffleFlag
 from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
 
 WAFFLE_FLAG_NAMESPACE = LegacyWaffleFlagNamespace(name='course_home')
 
-# .. toggle_name: course_home.course_home_mfe
-# .. toggle_implementation: ExperimentWaffleFlag
-# .. toggle_default: False
-# .. toggle_description: This experiment flag enables the use of the learning microfrontend (frontend-app-learning)
-#   as the default course frontend.
-# .. toggle_warnings: For this flag to have an effect, the LEARNING_MICROFRONTEND_URL setting must be configured, too.
-# .. toggle_use_cases: temporary
-# .. toggle_creation_date: 2020-05-29
-# .. toggle_target_removal_date: 2021-10-09
-# .. toggle_tickets: https://openedx.atlassian.net/browse/AA-117
-COURSE_HOME_MICROFRONTEND = ExperimentWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'course_home_mfe', __name__)
-
 COURSE_HOME_MICROFRONTEND_PROGRESS_TAB = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'course_home_mfe_progress_tab',  # lint-amnesty, pylint: disable=toggle-missing-annotation
                                                           __name__)
 
+COURSE_HOME_USE_LEGACY_FRONTEND = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'course_home_use_legacy_frontend', __name__)  # lint-amnesty, pylint: disable=toggle-missing-annotation
```

Comment:
So the lint-amnesty comments are to suppress old warning we had so we can get down to zero and then enforce a no pylint violations policy. In this case, I'd advocate for you to add the missing annotation here. The annotation should just look like what you deleted from `COURSE_HOME_MICROFRONTEND`, but updated to apply to this flag.

Also @ciduarte , the progress tab flag is looking mighty lonely without its annotation as well.